core/examples/luigi-example-angular/package.json

```diff
@@ -31,20 +31,21 @@
     "@angular-devkit/core": "0.8.9",
     "@angular-devkit/schematics": "0.8.9",
     "@angular/cli": "8.3.15",
-    "@angular/compiler-cli": "8.2.12",
+    "@angular/compiler-cli": "10.0.12",
```

Comment:
I get :  **`ERROR in Cannot read property 'map' of undefined`** when trying the _luigi-example-angular_ locally. 

Tried switching this line only back to **8.2.12** and installing packages again and running and it works as usual. 

Don't know if others have gotten any similar issues or whether we can make do without only this particular version bump, that would be good. 

Otherwise I would guess that probably bumping _"@angular/compiler-cli"_ up probably might require bumping up other packages under the same family _@angular/*_ , could be they're not totally backwards compatible with each other.

The Vue example works pretty well ✅

Reply:
the same

Reply:
The reason why it fails.. you upgrade angular/compiler-cli to version 10 without upgrading all other angular libs.
My suggestions:
dependencies:
```
    "@angular/animations": "^10.2.2",
    "@angular/common": "^10.2.2",
    "@angular/compiler": "^10.2.2",
    "@angular/core": "^10.2.2",
    "@angular/platform-browser": "^10.2.2",
    "@angular/platform-browser-dynamic": "^10.2.2",
    "@angular/router": "^10.2.2",
    "@luigi-project/client": "^1.5.0",
    "@luigi-project/core": "^1.5.0",
```

dev-dependencies:
```
    "@angular-builders/custom-webpack": "^10.0.1",
    "@angular-devkit/build-angular": "^0.1002.0",
    "@angular-devkit/core": "^10.2.0",
    "@angular-devkit/schematics": "^10.2.0",
    "@angular/cli": "10.2.0",
    "@angular/compiler-cli": "10.2.2",
    "@angular/language-service": "^10.2.2",
    "typescript": "^4.0.0",
```

These packages have to be upgraded, then it should work.

Reply:
did it in the past, unfortunately doesn't work and I'm getting the error:
"An unhandled exception occurred: Cannot read property 'Minus' of undefined
See "/private/var/folders/jq/0tgq0hxd703_gyplc7x0jt_h0000gn/T/ng-wWexNV/angular-errors.log" for further details."

Which is a known issue at version of @angular/compiler-cli higher than 10.0.12

Reply:
thanks. Works great!